prepare_kaufco.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
No need for this if it's actually empty.

Reply:
Fixed.